src/app/cases/page.tsx

```diff
@@ -44,73 +54,90 @@ export default function Page() {
-            defaultValue="Attorney/Interpreter"
-            options={['Attorney/Interpreter', 'Interpreter Only']}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, role: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="Anywhere"
-            // requires knowing user location
-            options={[
-              'Within 5 miles',
-              'Within 10 miles',
-              'Within 20 miles',
-              'Within 100 miles',
-              'Anywhere',
-            ]}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, distance: v as string })
-            }
-          />
-          <FilterDropdown
-            defaultValue="All countries"
-            multi
-            options={[
-              'All countries',
-              ...Array.from(
-                new Set(
-                  caseData
-                    .filter(c => c.country)
-                    .map(c => (c.country ? c.country : '')),
-                ),
-              ),
-            ]}
-            onChange={v =>
-              setCaseFilters({ ...caseFilters, countries: v as string[] })
-            }
-          />
-        </FiltersContainer>
-      </div>
+      <H2>Browse Available Cases</H2>
+      <FiltersContainer>
+        <FilterDropdown
+          placeholder={defaultFilterValues.remote}
+          multi
+          options={new Set(['Remote', 'In Person'])}
+          value={caseFilters.remote}
+          fullText="All Remote/In Person"
+          onChange={v => setCaseFilters({ ...caseFilters, remote: v })}
+        />
+        <FilterDropdown
+          placeholder={defaultFilterValues.role}
+          multi
+          options={new Set(['Interpreter', 'Attorney'])}
+          value={caseFilters.role}
+          fullText="All Interpreter/Attorney"
+          onChange={v => setCaseFilters({ ...caseFilters, role: v })}
+        />
+        {/* languages dropdown to implement later */}
+        <FilterDropdown
+          placeholder={defaultFilterValues.agency}
+          multi
+          options={
+            new Map(
+              caseData.map(c => [
+                c.adjudicating_agency,
+                parseAgency(c.adjudicating_agency),
+              ]),
+            )
+          }
+          value={caseFilters.agency}
+          onChange={v => setCaseFilters({ ...caseFilters, agency: v })}
+        />
+        <FilterDropdown
+          placeholder={defaultFilterValues.countries}
+          multi
+          options={
+            new Set(
+              caseData.filter(c => c.country).map(c => c.country),
+            ) as Set<string>
+          }
+          value={caseFilters.countries}
+          onChange={v => setCaseFilters({ ...caseFilters, countries: v })}
+        />
+        <ResetFilters onClick={() => resetFilters()}>
+          Reset Filters
+        </ResetFilters>
+      </FiltersContainer>
       <Body>
         <CardColumn>
```

Comment:
oop sorry missed this before, but can we also add the little text for the number of cases that fit the filters (i.e. "100 listings found" in this screenshot):

<img width="311" alt="Screenshot 2023-11-23 at 8 59 48 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/1f8f4f9d-c40e-40f1-a91d-7b12fc362b21">

Reply:
gonna draft something up but lowk this depends on design and should probably be iterated upon in a later sprint
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/e430ccb3-51aa-4579-b4fd-5af40a0c512f)

Reply:
cc-ing @davidqing6432 on this, but my personal feeling is to not make it fixed

Reply:
also yes, let's handle this in a different sprint tbh - you can leave this as is!

Reply:
oh i already made it... should i drop this change? currently it's fixed to better match the figma (if it's not, it will be lower)